frontend/packages/console-dynamic-plugin-sdk/src/api/types/statuses.ts

```diff
@@ -0,0 +1,28 @@
+export type StatusComponentProps = {
```

Comment:
As mentioned in my other comment https://github.com/openshift/console/pull/9521#discussion_r673268207 - let's avoid duplication; after copying/exposing relevant types via plugin SDK package, we should update the existing (shared, internal) code to use these types.

Reply:
For now we can have a copy of the type.